build.sbt

```diff
@@ -36,11 +35,11 @@ lazy val microservice = (project in file("."))
     addTestReportOption(Test, "test-reports")
   )
   .configs(IntegrationTest)
-  .settings(integrationTestSettings())
+  .settings(inConfig(IntegrationTest)(itSettings))
```

Comment:
For IT settings, follow this - https://github.com/hmrc/sbt-settings?tab=readme-ov-file#integration-tests